packages/frontend/src/connectors/readonly/index.ts

```diff
@@ -24,24 +25,18 @@ export class ReadonlyConnector extends Connector<
         super(config);
     }
 
+    public getChainIdFromState() {
```

Comment:
Why is this function a function? Would we ever need to access the chain id saved in the store from somewhere? And especially by accessing this connector?

Reply:
You are right, doesn't need to be on the class, I guess we can use it directly for now.

Reply:
fixed: https://github.com/carrot-kpi/v1-monorepo/pull/230/commits/32a0363aa3f971977a3dd77c1a7b5940fd1651fc